Core/PaymentMethodFactory.php

```diff
@@ -45,7 +45,7 @@ public static function create($sPaymentMethodType)
             case SofortPaymentMethod::getName(true):
                 return new SofortPaymentMethod();
             default:
-                throw new Exception("payment type not registered: $sPaymentMethodType");
+                throw new StandardException("payment type not registered: $sPaymentMethodType");
```

Comment:
Why is this not a lang key?

Reply:
This is never shown to the user – just like log messages – so it is not translated.